sklearn/decomposition/tests/test_nmf.py

```diff
@@ -943,20 +943,22 @@ def test_NMF_inverse_transform_W_deprecation():
         random_state=0,
         tol=1e-6,
     )
-    Xt = est.fit_transform(A)
+    X = est.fit_transform(A)
 
     with pytest.raises(TypeError, match="Missing required positional argument"):
```

Comment:
Your new helper raises a ValueError in that case.
```suggestion
    with pytest.raises(ValueError, match="Missing required positional argument"):
```
Or maybe change the helper to raise a TypeError. I don't have a preference.